lib/auth/types/openid/routes.js

```diff
@@ -81,6 +81,7 @@ module.exports = async function (pluginRoot, server, kbnServer, APP_ROOT, API_RO
     server.ext('onPreResponse', function(request, h) {
         // Make sure we only handle errors for the login route
         if (request.response.isBoom && request.path.indexOf(`${APP_ROOT}${routesPath}login`) > -1 && request.response.output.statusCode === 500) {
+            server.log(['error', 'security'], `Error occured while obtaining token: ${error}`);
```

Comment:
This will produce an error because `error` is not defined. It should be something like:
```
server.log(['error', 'security'], `Error occured while obtaining token: ${request.response}`);
```

Reply:
+1

Reply:
Please have a look at it now.